app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,5 @@
 	<string name="no_menu_text">No menu information 😮</string>
 	<string name="cornell_website_url">https://www.cornellappdev.com/</string>
```

Comment:
this `url` isn't named properly, i know it's out of scope for this PR, maybe we can open up an issue for it?

Reply:
what is the correct url?

Reply:
sorry poor wording on my part –– `string name` isn't named properly, `cornell_website_url` implies it would be `cornell.edu` or something